Makefile

```diff
@@ -5,20 +5,17 @@
 default: test
 
 test:
-	python -m unittest discover
-
-coverage:
-	coverage run -m unittest discover
-	coverage report -m
+	tox -e py27,py34
 
 pylint:
 	pylint edx_lint test setup.py
 
 clean:
 	-rm -rf *.egg-info
-	-rm -f *.pyc */*.pyc */*/*.pyc */*/*/*.pyc */*/*/*/*.pyc */*/*/*/*/*.pyc
-	-rm -rf __pycache__ */__pycache__ */*/__pycache__ */*/*/__pycache__ */*/*/*/__pycache__ */*/*/*/*/__pycache__
+	-find . -name '*.pyc' -delete
+	-find . -name '__pycache__' -delete
```

Comment:
Oh, I know why: because if the only files in there are .pyc files, then they were all deleted in the previous step :)
I'll make a better command.